qemu/tests/cfg/numa_memdev_options.cfg

```diff
@@ -0,0 +1,49 @@
+- numa_memdev_options:
+    type = numa_memdev_options
+    kill_vm_on_error = yes
+    numa_hardware_cmd = "numactl --hardware"
+    mem_fixed = 1536
+    mem_devs = "mem0 mem1"
+    backend_mem = memory-backend-ram
+    use_mem_mem0 = no
+    use_mem_mem1 = no
+    host-nodes_mem0 = 0
+    host-nodes_mem1 =1
+    size_mem0 = 512M
+    size_mem1 = 1024M
+    prealloc_mem0 = yes
+    prealloc_mem1 = yes
+    guest_numa_nodes = "node0 node1"
+    numa_memdev_node0 = mem-mem0
+    numa_memdev_node1 = mem-mem1
+    variants:
+        - policy_default:
+            policy_mem = default
+            del host-nodes_mem0
+            del host-nodes_mem1
+        - policy_bind:
+            policy_mem = bind
+        - policy_interleave:
+            policy_mem = interleave
+        - policy_preferred:
+            policy_mem = preferred
+    variants:
+        - numa_policy:
+        - numa_host-nodes:
+            no policy_default
+        - numa_mlock:
+            type = numa_memdev_mlock
+            mlock_cmd = "grep nr_mlock /proc/vmstat"
+            unevictable_cmd = "grep nr_unevictable /proc/vmstat"
+            realtime_mlock = on
+            start_vm = no
+        - numa_hugepage:
+            backend_mem = memory-backend-file
+            setup_hugepages = yes
+            mem-path = /mnt/kvm_hugepage
+            target_hugepages = 1500
+            variants:
+                - prealloc:
+                - no_prealloc:
+                    prealloc_mem0 = no
+                    prealloc_mem1 = no
```

Comment:
I prefer to move 'prealloc' and 'no_prealloc' outside so it could cover more scenarios including small pages.

Reply:
@huangyum no specific test cases except numa_hugepage from test cases, Are other test scenario (such as numa_policy,numa_mlock) affected by the prealloc?

Reply:
@jingzhao84 Yes, they could be.  Let's take advantage of automation to cover as many scenarios as we could. Thanks!

Reply:
Modified, Thanks!